src/components/animated/_glow.less

```diff
@@ -1,12 +1,7 @@
-:root {
-  --light: 80;
-  --threshold: 60;
-}
-
 .glowBorderRule(@color) {
   border-color: @color;
   background-color: rgba(@color, 0.2);
-  --switch: calc((var(--light) - var(--threshold)) * -100%);
+  --switch: calc((var(light) - var(threshold)) * -100%);
```

Comment:
Change var(light) to just @light. same for threshold. Also add switch to the variables like you did before.

Reply:
I realized those variables don't make sense. I was trying to change font color based on background but realized that is on user to modify based on their background as the button itself is translucent. Sorry for the confusion, it's my first time using less. Can you check now?